dbbackup/management/commands/dbbackup.py

```diff
@@ -50,18 +50,18 @@ def handle(self, **options):
             for database_key in database_keys:
                 database = settings.DATABASES[database_key]
                 self.dbcommands = DBCommands(database)
-                self.save_new_backup(database, database_key)
+                self.save_new_backup(database)
```

Comment:
Ah, I see, `database_name` is an unused argument in the function and you removed it. :+1: